configs/authentication/stackhawk-auth0.yml

```diff
@@ -0,0 +1,33 @@
+app:
+  applicationId: ${YOUR_APP_ID}
+  env: Auth0 
+  host: http://localhost:8000/ 
+  authentication:
+    loggedInIndicator: "2[0-9][0-9].*OK.*|.*3[0-9][0-9].*"
```

Comment:
We need to stop perpetuating these regex's they're extremely slow/broken in many cases. I'll fix the original example that started these.